src/kernels/linux-zen.sh

```diff
@@ -0,0 +1,108 @@
+# For build.sh
+mode_name="zen"
+mode_desc="Select and use the packages for the linux-zen kernel"
+
+# Kernel versions for default ZFS packages
+pkgrel="1"
+kernel_version="4.12.2-1"
```

Comment:
4.12.2 is in testing right now, so start building this packages when this hits extra?